src/irmin-chunk/_oasis

```diff
@@ -0,0 +1,35 @@
+OASISFormat: 0.4
```

Comment:
can you remove the oasis file? This is not needed as you converted the build to use jbuilder